LICENSE

```diff
@@ -1,13 +1,13 @@
 MIT License
 
-Copyright (c) 2021 Human Brain Project
+Copyright (c) 2021 CNRS
```

Comment:
we agreed now that we list here all currently involved institutes? or should we split for now as temporary place holder until we set up a proper openMINDS initiative?

Reply:
I thought it was all currently involved institutes on a per-module level (rather than all-of-openMINDS), i.e. as soon as we have a git commit from someone from U. Oslo in the v1 branch of this repository, we would add U. Oslo to the copyright statements...

I think this sends the right message as far as encouraging contributions from outside HBP.